mesheryctl/internal/cli/root/system/update.go

```diff
@@ -122,14 +184,42 @@ var updateCmd = &cobra.Command{
 				if err != nil {
 					return err
 				}
+
+				err = utils.ApplyOperatorManifest(client, true, false)
+
+				if err != nil {
+					return err
+				}
 			}
+
+			skipUpdateFlag = true
```

Comment:
Why do we need this flag here?

Reply:
This is to avoid redundant updates on `start`(when restarted) since it has already been updated.

Reply:
But system update does not call system start right?

Reply:
Same changes that I proposed here - https://github.com/layer5io/meshery/pull/3001#issuecomment-848857022

Reply:
`update` does call `restart`, which calls `start`(for docker).  
Maybe for kubernetes, I could remove it.

Reply:
We do not need to restart for docker right? Just updating the container images should work

Reply:
I called `restart` in docker, seeing how it was called in kubernetes.  
Should I remove it?

Reply:
@navendu-pottekkat I have re worked the behaviour when platform: docker and removed `restart` from it.